gef-ui/server/controllers/application-details/index.test.js

```diff
@@ -216,6 +216,11 @@ describe('controllers/application-details', () => {
           status: DEAL_STATUS.CANCELLED,
           expectObjectContaining: { applicationStatus: DEAL_STATUS.CANCELLED },
         },
+        {
+          template: `application-preview`,
+          status: DEAL_STATUS.PENDING_CANCELLATION,
+          expectObjectContaining: { applicationStatus: DEAL_STATUS.PENDING_CANCELLATION },
+        },
```

Comment:
Wow isnt this nice and easy to update, who did this lovely refactor

Reply:
Amazing refactor 😉